mission:
  rooms_order:
    - KITCHEN
    - GUESTROOM


rooms:
  - name: BEDROOM
    qr_expected: "ROOM=BEDROOM"
    entry_goal:
      x: -5.96
      y: 0.07
      yaw: 0.02

      
      
  - name: LIVINGROOM
    qr_expected: "ROOM=LIVINGROOM"
    entry_goal:
      x: -6.10
      y: 2.12
      yaw: -0.02

      
      
  - name: CORRIDOR
    qr_expected: "ROOM=CORRIDOR"
    entry_goal:
      x: -1.35
      y: 1.10
      yaw: 0.0

      
      
  - name: DININGROOM
    qr_expected: "ROOM=DININGROOM"
    entry_goal:
      x: 3.05
      y: 1.11
      yaw: -1.57

      
      
  - name: GUESTROOM
    qr_expected: "ROOM=GUESTROOM"
    entry_goal:
      x: 1.25
      y: 3.48
      yaw: -0.03

      
      
  - name: KITCHEN
    qr_expected: "ROOM=KITCHEN"
    entry_goal:
      x: 6.09
      y: -1.38
      yaw: -0.02



